client/app/dbaas/logs/detail/logs-helper.service.js

```diff
@@ -118,8 +118,8 @@ class LogsHelperService {
    * return true if account is of type basic, false otherwise
    * @param {accountDetails} account
    */
-  static isBasicOffer(account) {
-    return !account.offer.reference.startsWith('logs-pro');
+  isBasicOffer(account) {
```

Comment:
If this method is no more a static method then you have to update wherever it's called.

```sh
$ git grep -i 'constructor.isBasicOffer'
client/app/dbaas/logs/detail/home/logs-home.controller.js:    if (this.LogsHelperService.constructor.isBasicOffer(this.account.data)) {
```

Eg: `this.LogsHelperService.constructor.isBasicOffer` become `this.LogsHelperService.isBasicOffer`